<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_page</class>
 <widget class="QMainWindow" name="settings_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pics/official_icon.png</normaloff>:/pics/official_icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,25,25);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>200</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="user_symbol">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>130</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/pics/USER.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="settings_msg_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>561</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt  &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Settings:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>220</width>
      <height>196</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/pics/official_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>3</width>
      <height>664</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="log_out_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>220</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>LOG OUT</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username_msg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>YOUR CURRENT USERNAME IS </string>
    </property>
   </widget>
   <widget class="QLabel" name="username_msg_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>115</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>YOU CAN CHANGE IT HERE &gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="change_name_btn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>114</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="password_change_msg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>YOU CAN ALSO CHANGE YOUR PASSWORD OVER HERE:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="current_pass">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>290</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_text">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font:bold 8pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>CURRENT PASSWORD</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="new_username">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_text_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font:bold 8pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>NEW PASSWORD</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="new_pass">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="confirmpassword">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	border: 1px solid white;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_text_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font:bold 8pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>CONFIRM NEW PASSWORD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_password_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>487</y>
      <width>271</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(81, 139, 141);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CHANGE PASSWORD </string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="change_username_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>221</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CHANGE USERNAME </string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back_to_home_btn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>HOME →</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>icon</zorder>
   <zorder>username</zorder>
   <zorder>line_2</zorder>
   <zorder>user_symbol</zorder>
   <zorder>settings_msg_2</zorder>
   <zorder>line</zorder>
   <zorder>log_out_btn</zorder>
   <zorder>username_msg</zorder>
   <zorder>username_msg_2</zorder>
   <zorder>change_name_btn</zorder>
   <zorder>password_change_msg</zorder>
   <zorder>current_pass</zorder>
   <zorder>password_text</zorder>
   <zorder>new_username</zorder>
   <zorder>password_text_2</zorder>
   <zorder>new_pass</zorder>
   <zorder>confirmpassword</zorder>
   <zorder>password_text_3</zorder>
   <zorder>change_password_btn</zorder>
   <zorder>change_username_btn</zorder>
   <zorder>back_to_home_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
